apps/debugger/webpack.config.js

```diff
@@ -1,17 +1,27 @@
 const nxWebpack = require('@nrwl/react/plugins/webpack')
+const TerserPlugin = require('terser-webpack-plugin')
 
 module.exports = config => {
-    const nxWebpackConfig = nxWebpack(config)
+  const nxWebpackConfig = nxWebpack(config)
+  const productionConfig = process.env.NODE_ENV === 'production' ? {
+    mode: 'production',
+    devtool: 'source-map',
+    optimization: {
+      minimize: true,
+      minimizer: [new TerserPlugin()]
+    }
+  } : {}
 
-    return {
-        ...nxWebpackConfig,
-        node: {
-            fs: 'empty',
-            tls: 'empty',
-            readline: 'empty',
-            net: 'empty',
-            module: 'empty',
-            child_process: 'empty'
-        }
+  return {
+    ...nxWebpackConfig,
+    ...productionConfig,
```

Comment:
Should we put a check on return object to return `productionConfig` only if environment is production? That can be better from returning even `{}`

Reply:
Ok. I'll make the change.